### PR TITLE
fix: Get stylelint working again

### PR DESCRIPTION
**Pull request by ccorda**
The latest version of the vscode extension requires an upgrade to stylelint, which requires some configuration to enable scss

### CODE REVIEW

**Comment by ccorda** [comment]
I based this on the next 12 PR, one that is merged, then this can go on top.

File: website/package.json [hunk 1]
```diff
@@ -21,39 +21,40 @@
...
+    "prettier": "^2.4.1",
+    "stylelint": "^14.1.0",
     "stylelint-config-idiomatic-order": "^8.1.0",
-    "stylelint-config-prettier": "^8.0.2",
-    "stylelint-config-standard": "^22.0.0",
-    "stylelint-prettier": "^1.2.0",
-    "stylelint-scss": "^3.20.1"
+    "stylelint-config-prettier": "^9.0.3",
+    "stylelint-config-standard": "^24.0.0",
+    "stylelint-config-standard-scss": "^2.0.1",
```
Comment (chrisherold) [inline website/package.json:55]: just updated to 3.0.0 three days ago which uses `stylelint-config-standard@24.0.0` and doesn't have any breaking changes so seems like a good update

Comment (ccorda) [reply website/package.json:55]: does this mean we can just use a single dep in here (and remove `stylelint-config-standard`)?

Comment (chrisherold) [reply website/package.json:55]: yep, its in `stylelint-config-standard-scss`'s dependencies

**Review by chrisherold — APPROVED** [review]
lgtm

**Pull request merged**